ckanext/saml2auth/spconfig.py

```diff
@@ -23,8 +27,16 @@ def get_config():
     remote_cert = \
         ckan_config.get(u'ckanext.saml2auth.idp_metadata.remote_cert')
 
+    issuer = ckan_config.get(u'ckanext.saml2auth.issuer', u'urn:mace:umu.se:saml:ckan:sp')
+    response_signed = asbool(ckan_config.get(u'ckanext.saml2auth.want_response_signed', True))
+    assertion_signed = asbool(ckan_config.get(u'ckanext.saml2auth.want_assertions_signed', False))
+    any_signed = asbool(ckan_config.get(u'ckanext.saml2auth.want_assertions_or_response_signed', False))
+    key_file = ckan_config.get(u'ckanext.saml2auth.key_file_path', None)
+    cert_file = ckan_config.get(u'ckanext.saml2auth.cert_file_path', None)
+    attribute_map_dir = ckan_config.get(u'ckanext.saml2auth.attribute_map_dir', None)
+
```

Comment:
Please document all additional fields in the README.rst and add tests.

Reply:
Done @duskobogdanovski